template.go

```diff
@@ -0,0 +1,32 @@
+package cloudconfig
+
+import (
+	"github.com/giantswarm/microerror"
+
+	"github.com/giantswarm/k8scloudconfig/v_0_1_0"
+)
+
+type version string
+
+const (
+	V_0_1_0 version = "v_0_1_0"
```

Comment:
Maybe we want to add `Latest` alias. Uncertain.

Reply:
A latest alias could be useful for local development and test installations. IMO for production clusters it should always come from the cluster custom object.

Reply:
I think we can add this later if we really want to. The update story is super big and I am still trying to wrap my head around. I want to make baby steps and get a look and feel. Thus I would keep the latest version out of this PR at least. Lets iterate. <3